src/prommis/uky/uky_flowsheet.py

```diff
@@ -727,8 +1335,12 @@ def set_scaling(m):
 def set_operating_conditions(m):
     """
     Set the operating conditions of the flowsheet such that the degrees of freedom are zero.
+
+    Args:
+        m: pyomo model
     """
-    eps = 1e-7 * units.mg / units.L
+    # Epsilon represents near-zero component concentrations
+    eps = 1e-8 * units.mg / units.L
```

Comment:
I suggest making this a module level constant at the top of the file so it is easier to find and change.